<launch>

  <!-- Urg Node -->
  <!-- <node pkg="tms_ss_pot" type="change_nan" name="change_nan">
      <remap from="/most_intense" to="/hsrb/base_scan"/>
      <remap from="/LaserTracker" to="/LaserTracker1"/>
  </node> -->

  <param name="/use_sim_time" value="false"/>

  <!-- Tracker and Visualizer -->
  <node pkg="tms_ss_pot" name="urg_single_tracker" type="urg_single_tracker" output="screen">
    <remap from="/tracking_points" to="/tracking_points2"/>
  </node>
  <node pkg="tms_ss_pot" name="visualize" type="visualize">
    <remap from="/tracking_points" to="/tracking_points2"/>
  </node>

  <!-- TF -->
  <node pkg="tf" type="static_transform_publisher" name="world_2_map" args="0 0 0
  0 0 0 /world_link /map 100"/>

  <!-- Map server -->
  <arg name="map_file" default="$(find tms_ss_pot)/maps/map_bsen_p/map.yaml"/>
  <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)">
  <param name="frame_id" value="map" />
  </node>

  <!-- RViz -->
  <node name="rviz_tracker" pkg="rviz" type="rviz" args="-d $(find tms_ss_pot)/rviz/tracker_config.rviz" respawn="true" />

</launch>
